Flow.Launcher.Core/Configuration/Portable.cs

```diff
@@ -126,13 +124,16 @@ public void CreateUninstallerEntry()
         {
             var uninstallRegSubKey = @"Software\Microsoft\Windows\CurrentVersion\Uninstall";
             // NB: Sometimes the Uninstall key doesn't exist
-            using (var parentKey =
-                RegistryKey.OpenBaseKey(RegistryHive.CurrentUser, RegistryView.Default)
-                    .CreateSubKey("Uninstall", RegistryKeyPermissionCheck.ReadWriteSubTree)) {; }
+            RegistryKey
+                .OpenBaseKey(RegistryHive.CurrentUser, RegistryView.Default)
+                .CreateSubKey("Uninstall", RegistryKeyPermissionCheck.ReadWriteSubTree)
+                .Dispose();
```

Comment:
We should stick to using ()

Reply:
It's a matter of preference, but I don't really like the empty using statements... the empty `{ }` feel like noise to me, always catching my eye when I'm scrolling through the file. But if the team prefers them that way, it's ok with me.

Reply:
BTW, this specific step (lines 127-130) is probably here by accident... it creates a key named "Uninstall" in `HKEY_CURRENT_USER`...